lisdf/components/base.py

```diff
@@ -0,0 +1,23 @@
+#! /usr/bin/env python3
+# -*- coding: utf-8 -*-
+# File   : base.py
+# Author : Jiayuan Mao
+# Email  : maojiayuan@gmail.com
+# Date   : 03/23/2022
+#
+# This file is part of lisdf.
+# Distributed under terms of the MIT license.
+
+
+class StringConfigurable(object):
```

Comment:
Thanks. Yes, this is left as a TODO. And in general, maybe we will gradually add implementations to the system. But if I leave them as abstract it will trigger errors when I instantiate class? What do you think is the best way...

Reply:
Just make this class ABC so we can't instantiate it and add some TODOs so we will remember and I think it's ok!